scripts/sct_check_dependencies.py

```diff
@@ -27,7 +27,7 @@ def __init__(self):
 
 
 import sys, io, os, commands, platform, importlib
-
+from cStringIO import StringIO
```

Comment:
`cStringIO` is gone in Python3, so I'd suggest to simply use `io.BytesIO`. See later.

Reply:
Will fix it.